src/qml/imports/Theme/QfScrollBar.qml

```diff
@@ -0,0 +1,93 @@
+import QtQuick
+import QtQuick.Controls.impl
+import QtQuick.Templates as T
+import Theme
+
+T.ScrollBar {
+  id: control
+
+  property color color: Theme.mainColor
+  readonly property real _maxSize: 8
+  readonly property real _minSize: 4
+
+  width: horizontal ? parent.width : _maxSize
+  height: horizontal ? _maxSize : parent.height
+  visible: control.policy !== T.ScrollBar.AlwaysOff
+  anchors.right: parent.right
+  background: Rectangle {
+    id: background
+    radius: _minSize
+    color: Theme.darkTheme ? "#2c2c2c" : "#aaffffff"
```

Comment:
Can we use Theme colors here? If we're missing a color component, better add to Theme.